hub/api/dataset_utils.py

```diff
@@ -0,0 +1,47 @@
+from typing import Union
+from hub.codec import PngCodec, Lz4, NumPy, Zstd, JpegCodec, WebPCodec
+from hub.codec import BaseImgCodec, BaseNumCodec
+
+compression_map = {
+    PngCodec().__name__: PngCodec,
```

Comment:
the way i had this was kinda thrown together, instead of this dictionary, can you create an `__init__.py` file under `hub.codec` and within there do something like:

```
from .jpg import JPG
# ... for every codec
```

and instead of a map, you can just do:

```
from hub import codec

compression = codec["jpg"]()
compression.compress(...)
```

Reply:
sorry, i just saw that you already have the `hub.codec` init file :) this makes it even easier!